calculator-app/src/App.js

```diff
@@ -1,25 +1,23 @@
-import logo from './logo.svg';
-import './App.css';
-
+import React,{useState, createContext } from 'react';    // importing Hook and React
+import './App.css';   // Importing App.css for accessing class
+import Editbox from './Editbox'     // Import Editbox.js 
+import Buttonnumbers from './Buttonnumbers'   // Import buttonnumbers.js 
+export const ButtonContext = createContext();   // to create context for passing the state
```

Comment:
please explain me the reason for export file to be write in a upside of the page